crates/rolldown_error/src/event_kind.rs

```diff
@@ -14,6 +14,7 @@ pub enum EventKind {
   // TODO remove following kinds
   NapiError,
   IoError,
+  AnyhowError,
```

Comment:
We use anyhow to represent program errors and `BuildError` as errors related to bundling.

So anyhow Error should not be encapsulated by BuildError.